src/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.cpp

```diff
@@ -127,6 +127,31 @@ tnsr::abb<DataType, Dim, Frame> compute_derivatives_of_spacetime_metric(
   return spacetime_deriv_spacetime_metric;
 }
 
+template <size_t SpatialDim, typename Frame, typename DataType>
+tnsr::a<DataType, SpatialDim, Frame> compute_spacetime_normal_one_form(
+    const Scalar<DataType>& lapse) noexcept {
+  tnsr::a<DataType, SpatialDim, Frame> normal_one_form{};
+  get<0>(normal_one_form) = -get<>(lapse);
+  for (size_t i = 0; i < SpatialDim; ++i) {
+    normal_one_form.get(i + 1) = make_with_value<DataType>(get<>(lapse), 0.);
+  }
+  return normal_one_form;
+}
+
+template <size_t SpatialDim, typename Frame, typename DataType>
+tnsr::A<DataType, SpatialDim, Frame> compute_spacetime_normal_vector(
+    const Scalar<DataType>& lapse,
+    const tnsr::I<DataType, SpatialDim, Frame>& shift) noexcept {
+  tnsr::A<DataType, SpatialDim, Frame> T_I{};
+
+  get<0>(T_I) = 1. / get<>(lapse);
+  for (size_t i = 0; i < SpatialDim; i++) {
+    T_I.get(i + 1) = -shift.get(i) * get<0>(T_I);
```

Comment:
Question: when do we prefer `get<i>(bla)` versus `bla.get(i)`?

Reply:
`get<..>(blah)` is one at compile time, so if `i` is a compile time constant we prefer `get<i>`, hence the use of `get<0>`, however `i` is determined at runtime here so `.get(i) `is used.